Cargo.toml

```diff
@@ -11,8 +11,10 @@ description = "A more memory efficient replacement for serde_json::Value"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [features]
 tracing = ["mockalloc/tracing"]
+thread_unsafe = ["ctor"]
 
 [dependencies]
+hashbrown = "0.13.2"
```

Comment:
This is unnecessary, the `std::collections::Hash**` types already used hashbrown.

Reply:
I needed it since `get_or_insert_with` is still not available in `std::collections::Hash**`

Reply:
It's still only available on nightly 
https://doc.rust-lang.org/std/collections/hash_set/struct.HashSet.html#method.get_or_insert_with